### ISSUE TITLE
Message stuck in "Queued" status

### ISSUE THREAD

**Issue by timoleo23** [OfficeDev/microsoft-teams-apps-company-communicator#1065]
Message sent remain in 'queued' status.
Draft message are correctly sent into the general channel but stay blocked when sending to the targeted list of users.

We already tried the steps described in this issue
https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/issues/628
- Go to Service Bus Namespace
- Navigate to the queues (**-prep and **-send)
- For each queue, do the following:
- Go to Service Bus Explorer (preview)
- Click on “Receive” tab
- Click “Receive” for each message still active.
- Sync the *-prep-function.
- Re-start the prep-function and try sending messages.  

There is no error generated on the Azure send or prep functions.
We don't see errors on App Insights either.

We're using CCv5.0

**Comment by gsv022** [comment]
Hi @timoleo23 ,
Thanks for raising the query. Could you please share the screenshot of deployment status of Prep-function.
Go to azure portal-->Go to resource group(Where the deployment has been done)-->Select  Prep-function from list of resources-->Go to Deployment Center-->Click on logs

**Comment by gsv022** [comment]
Hi @timoleo23 ,
Please share your outlook id(mail id) to discuss the issue internally.

**Comment by gsv022** [comment]
Hi @timoleo23 ,
During the call, it was discussed that the status of the messages was showing up as 'queued' in two instances one in v5.0 (Prod) and another in v5.3 (Dev) version.
•	As a part of troubleshooting, we noticed that pipeline deployment using terraform templates has been followed instead of regular approach due to company policies. 
•	Each resource name is modified as per their requirement.
•	For v5.3 when the UI is shown up, the messages which are in queued state started getting delivered automatically.
•	The cause for this behavior was there an "**always on**" parameter present in the general settings of functions. It was set to off state. Due to this function is going to idle state and not delivering the message.
•	We recommended to enable the "**always on**" parameter for both the environments and recommended to upgrade the Prod instance of deployed CC app from v5.0 to at least v5.3.
